src/main/java/org/opensearch/knn/index/codec/KNN80Codec/KNN80DocValuesProducer.java

```diff
@@ -75,6 +75,9 @@ public KNN80DocValuesProducer(DocValuesProducer delegate, SegmentReadState state
                 continue;
             }
             List<String> engineFiles = KNNCodecUtil.getEngineFiles(knnEngine.getExtension(), field.name, state.segmentInfo);
+            if (engineFiles.isEmpty()) {
```

Comment:
Can we change description to account for this fix?

Reply:
Created #2147 to address this change in main. I will rebase once it is merged.